src/mtcp/mtcp_util.h

```diff
@@ -162,7 +162,10 @@ void mtcp_printf(char const *format, ...);
 ssize_t mtcp_read_all(int fd, void *buf, size_t count);
 int mtcp_readfile(int fd, void *buf, size_t size);
 void mtcp_skipfile(int fd, size_t size);
-unsigned long mtcp_strtol(char *str);
+unsigned long mtcp_strtol(const char *str);
+unsigned long long mtcp_strtoll(const char *str);
+void mtcp_utoa(char* buffer, unsigned int n);
+void mtcp_itoa(char* buffer, int n);
```

Comment:
Even though you don't need it here, I've sometimes missed:
> mtcp_strcmp

We have 'mtcp_strcpy', but not 'mtcp_strcmp'.  It's often a convenient utility.